packages/react/src/components/Storage/File-Uploader/FileUploader.tsx

```diff
@@ -0,0 +1,87 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
Can we enable this lint? I do think it's useful for maintainability purposes

Reply:
OK, this was mostly for the props I haven't implemented yet. I'll add them to a console.log add disable that warning for now. I suppose the other way is to remove them completely, but I don't want to do that right now.